### DISCUSSION TITLE
How to get login cookie in Jython script and call it in related script

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#735] [Q&A] [answered]
测试场景是登录获取cookie，然后在下一个脚本中使用cookie维持登录状态
The test scenario is to log in to get the cookie, and then use the cookie to maintain the login state in the next script

**Comment by imbyungjun** [comment, marked as answer]
You can find a jython login scenario sample code [here](https://github.com/naver/ngrinder/blob/master/script-sample/test-with-login/login.py).

If you want to maintain the cookie in **another** script(=another performance test), you have to store the cookie value to somewhere and then, set the cookie before run the test.

**Reply by zhougy1989** [reply]
thanks,it work
